### PR TITLE
Feat/rs mongo support

### PR DESCRIPTION
**Pull request by mohdashraf010897**
**PR Type** `Feature`

**Description** 
- Add support for `mongodb` for ReactiveSearch

**Libraries Affected**
- [x] React
- [x]  Vue

[🎥  Loom: React](https://www.loom.com/share/d9d8a9b8a494437d88045e4611c8ac31)

---
Updated Demos:
- [ES Demo (React + Vue)](https://www.loom.com/share/8f1324904c94490f88510dca61fd3c07)
- [MONGO Demo (React + Vue)](https://www.loom.com/share/e76e7bdac2a944e0b577a9d570ff2727)
---

[Notion Link](https://www.notion.so/appbase/Reactivesearch-Mongo-DB-Support-1ef6e865b5cf4204aaae338d22afa288)

--- 
https://github.com/appbaseio/appbase-js/pull/84
https://github.com/appbaseio/reactivecore/pull/111
https://github.com/appbaseio/reactivecore/pull/112
---

TODO: publish `appbase-js` and upgrade versions in `reactivesearch` libraries: `web` and `vue`

### CODE REVIEW

File: packages/web/src/components/basic/ReactiveBase.js [hunk 1]
```diff
@@ -108,7 +110,7 @@ class ReactiveBase extends Component {
 			...props.appbaseConfig,
 		};
 		const config = {
-			url: props.url && props.url.trim() !== '' ? props.url : 'https://scalr.api.appbase.io',
+			url: props.url,
```
Comment (bietkul) [inline packages/web/src/components/basic/ReactiveBase.js]:
```suggestion
			url: props.url && props.url.trim() !== '' ? props.url : '',
```

**Review by bietkul — CHANGES_REQUESTED** [review]
@mohdashraf010897 You didn't make changes for Vue?

File: packages/web/src/components/basic/ReactiveBase.js (hunk 1 above)
Comment (mohdashraf010897) [reply packages/web/src/components/basic/ReactiveBase.js]: fixed here 👉 https://github.com/appbaseio/reactivesearch/pull/1810/commits/c4f50a6a4916d59045b0d493f530e0244a1e12be

**Comment by mohdashraf010897** [comment]
> @mohdashraf010897 You didn't make changes for Vue?

Yes, once you approve for React, i would.

**Review by bietkul — APPROVED** [review]

**Comment by bietkul** [comment]
@mohdashraf010897 Why is it not throwing `URL` nor defined error?
alt d

It seems the URL is wrong at here (`https` is being used twice)
alt d

**Review by bietkul — APPROVED** [review]

**Review by bietkul — APPROVED** [review]

**Pull request merged**